test/whitespace/functions.unit

```diff
@@ -165,4 +165,12 @@ main() {
 >>> generic function typed parameter
```

Comment:
Done, but it turns out this doesn't work—the logic above doesn't see the comment as a statement. How should I check for something like that?